android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -187,10 +189,16 @@ public void onClick(View v) {
             Timber.d(date+"\n"+endTime+"\n"+startTime);
 
         }
-        summary.setText(session.getSummary());
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
```

Comment:
Yes, @mananwason I understand that I have removed unnecessary extra check.
I am stuck with my OS(crashed), will correct it and push in sometime.